### PR TITLE
HDDS-2744. shell/BucketCommands.md translation

### PR DESCRIPTION
**Pull request by iamabug**
## What changes were proposed in this pull request?

one .md file

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2744

## How was this patch tested?

hugo server -D

### CODE REVIEW

**Comment by iamabug** [comment]
@xiaoyuyao Could you help review this ? Thanks.

PS, sorry for the mistyped commit message, no sure how to correct it.

**Comment by xiaoyuyao** [comment]
You can do a local "git rebase -i" and reword your commit message and then force push to iamabug:HDDS-2744.

File: hadoop-hdds/docs/content/shell/BucketCommands.zh.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: 桶命令
+summary: 桶命令帮助你管理桶的生命周期
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/shell/BucketCommands.zh.md]: 用桶命令管理桶的生命周期

File: hadoop-hdds/docs/content/shell/BucketCommands.zh.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+### 创建
+
+用户使用 `bucket create` 命令来创建桶。
+
+***参数：***
+
+| 参数名                      |  说明                                |
+|--------------------------------|-----------------------------------------|
+| -g, \-\-enforcegdpr            | 可选，如果设置为 true 则创建符合 GDPR 规范的桶，设置为 false 或不指定则创建普通的桶|
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/shell/BucketCommands.zh.md]: There is a --bucketKey option missing from the original English document. I've opened HDDS-2937 to fix it.

**Comment by xiaoyuyao** [comment]
Thanks @iamabug  for the update. LGTM, +1 I will merge it shortly.

**Pull request merged**